Lib/fontTools/feaLib/variableScalar.py

```diff
@@ -84,11 +84,17 @@ def default(self):
         return self.values[key]
 
     def value_at_location(self, location):
-        loc = location
+        if isinstance(location, dict):
+            loc = Location(location)
```

Comment:
this `Location` might as well be an actual type, instead of a function disguised as one